content/about/contribute/github/index.md

```diff
@@ -28,30 +28,78 @@ The documentation is published under the [Apache
 
 ## How to contribute
 
-There are two ways you can contribute to the Istio documentation:
+There are three ways you can contribute to the Istio documentation:
 
 * If you want to edit an existing page, you can open up the page in your
   browser and select the **Edit This Page on GitHub** option from the gear menu
   at the top right of each page. This takes you to GitHub to edit and
   submit the changes.
 
-* If you want to work on the site in general, you must create a fork of the
-  repository. Click the button below to visit the GitHub repository. Then, you
-  must click the **Fork** button in the upper-right corner of the screen to
-  create a copy of our repository in your GitHub account. Create a clone of
-  your fork and make any changes you want. When you are ready to send those
-  changes to us, push the changes to your fork, go to the index page for your
-  fork, and click **New Pull Request** to let us know about it.
+* If you want to work on the site in general, follow the steps in our
+  [How to add content section](#add).
 
-<a class="btn btn-istio"
-href="https://github.com/istio/istio.io/">Browse this site's source
-code</a>
+* If you want to review an existing pull request (PR), follow the steps in our
+  [How to review content section](#review)
 
 Once your changes are merged, they show up immediately on
 `preliminary.istio.io`. However, the changes only
 show up on `istio.io` the next time we produce a new
 release, which happens around once a quarter.
 
+### How to add content {#add}
+
+To add content you must create a fork of the repository and a PR from
+your fork to the docs main repository. The following steps describe the
+process:
+
+1.  Click the button below to visit the GitHub repository.
+
+    <a class="btn btn-istio"
+    href="https://github.com/istio/istio.io/">Browse this site's source
+    code</a>
+
+1.  Click the **Fork** button in the upper-right corner of the screen to
+    create a copy of our repository in your GitHub account.
+
+1.  Create a clone of your fork and make any changes you want.
+1.  When you are ready to send those changes to us, push the changes to your
+    fork.
+1.  Go to the index page for your fork, and click **New Pull Request** to let
+    us know about it.
+
+### How to review content {#review}
+
+If your review is small, simply comment on the PR directly. If you review the
+content in detail, follow these steps:
+
+1.  Leave a comment on the PR with the text `/hold`. This command prevents the
```

Comment:
this step is overkill imo :)

Reply:
That is a fair complaint.  Typically how we handle this in the code base, is if a patch merges prematurely, the onus is on the committer to fix the problems found post merge.  If the patch problems are egregious, the onus is on the reviewer to revert the commit and trigger the commiter to resubmit their work.

Perhaps we should have different set of workflows for docs vs code, as code reviews rarely merge ahead of a very solid review (and there are 30+ checks).  Docs are different - less checks, and more human work needed during the review process.

Cheers
-steve

Reply:
I agree. I don't think the right approach to revert someone's contribution to our content. I much rather iterate on it because, unlike in code, there are several ways of writing the same idea. My proposal here is to add provide a transparent way to ensure feedback is applied to reduce the quality-debt of the content. So far, only @vadimeisenbergibm has followed the process you describe @sdake. In my experience, it is hard enough to get the initial contribution to the docs. After most contributions, authors never revisit the content unless there was a code change that requires them to and, even then, that doesn't always happen. With this process, we can ensure, at the very least, the technical accuracy and basic clarity of content.